dev/tests/integration/testsuite/Magento/Framework/Backup/DbTest.php

```diff
@@ -41,6 +41,8 @@ public static function setUpBeforeClass(): void
      */
     public function testBackupAndRollbackIncludesCustomTriggers()
     {
+        $this->markTestIncomplete('PR: https://github.com/mage-os/mageos-magento-zend-db/pull/1 needs to be merged first.');
```

Comment:
The referenced PR is merged now.

Reply:
We would actually need to perform our own release with a tag for zend-db package and include it in Mage-OS.  
We can also lock thick package on a main branch - however, I would prefer to have it released.

Reply:
As discussed during the call, we need to keep this test disabled for now.
To properly test cross-repository dependencies, we need to create a full build before running the full integration testsuite, but that capability currently does not yet exist.
Until then we keep the test disabled.